src/utils/dispute-utils.js

```diff
@@ -84,21 +83,26 @@ function overrideVoidedDispute(dispute, voidedDispute) {
  * Construct dispute timeline for the given dispute
  * @param {Object} dispute The dispute to get the timeline from
  * @param {Object} courtConfig The court configuration
+ * @param {Number} currentTerm The court current term
  * @returns {Array} The timeline of the given dispute
  */
-export function getDisputeTimeLine(dispute, courtConfig) {
+export function getDisputeTimeLine(dispute, courtConfig, currentTerm) {
```

Comment:
I'm not sure we want to make this change. It's not as if a dispute is still open if the term hasn't adjusted—it definitely has closed / moved on to the next round, but the internal state is just not updated yet.

Reply:
Could you extend on this? I'm not totally sure i follow. 

Note that we are tracking the expected current term now as opposed to the internal court contract term as explained here https://github.com/aragon/court-dashboard/pull/335#discussion_r428656118